docs/src/user-guide/command-line-interface.md

```diff
@@ -241,71 +229,88 @@ npx eslint --rule 'jquery/dollar-sign: error'
 
 This option allows you to specify another directory from which to load rules files. This allows you to dynamically load new rules at run time. This is useful when you have custom rules that aren't suitable for being bundled with ESLint.
 
-Example:
+The rules in your custom rules directory must follow the same format as bundled rules to work properly. You can also specify multiple locations for custom rules by including multiple `--rulesdir` options.
```

Comment:
`--rulesdir` is missing "Argument Type" and "Multiple Arguments: No"

Reply:
sure, added in 4e49c050d